google-cloud-firestore/samples/query_data.rb

```diff
@@ -183,6 +188,17 @@ def invalid_range_query project_id:, collection_path: "cities"
   # [END firestore_query_filter_range_invalid]
 end
 
+def firestore_query_filter_compound_multi_ineq project_id:, collection_path: "cities"
+  # project_id = "Your Google Cloud Project ID"
+  # collection_path = "cities"
+
+  firestore  = Google::Cloud::Firestore.new project_id: project_id
+  cities_ref = firestore.col collection_path
+  # [START firestore_query_filter_compound_multi_ineq]
```

Comment:
The other samples below include the setting of `cities_ref` within the sample's START..END block. Are you sure this one shouldn't include it?

Reply:
I will test it. Samples above this point have `cities_ref` outside, and below have it inside.